build-support/scripts/dn

```diff
@@ -1,8 +1,28 @@
 #!/usr/bin/env bash
 
-docker stop te1 te2 sm1 ad1
-docker network rm nuodb-net
-##rm -fr vol1 vol2
+COMMAND="docker exec ad1 nuocmd check database --db-name test --check-running"
+if ! $COMMAND 2>/dev/null; then
```

Comment:
The `nuocmd check` output nothing on success. We should not suppress the output since we want to know why the database did not come up.

Reply:
The check is to avoid running the docker clean up. Any command will work although, it only checks for ad1 container. If for some reason other containers are still running, it wont stop them.
Maybe I need a better approach to this.

Reply:
Why not just unconditionally run `docker rm -f` on the containers, if you just want to make sure they are all cleaned up?

Reply:
That is what I had, but it wasn't pretty. Hopefully my next commit is the last change.